### PR TITLE
Feature/add country packagename to iab

### PR DESCRIPTION
**Pull request by GustavoRSSilva**
countryCode and the packagename added to the event Buy.

### CODE REVIEW

File: contracts/AppCoins.sol [hunk 1]
```diff
@@ -17,9 +17,9 @@ contract AppCoins is ERC20Interface{
     // Public variables of the token
```
Comment (DioPires) [inline contracts/AppCoins.sol:17]: #### Tests:
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline contracts/AppCoins.sol:17]: #### Tests:
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

File: contracts/AppCoinsIAB.sol [hunk 2]
```diff
@@ -8,11 +8,10 @@ contract AppCoins {
 
 contract AppCoinsIABInterface {
     function division(uint numerator, uint denominator) public view returns (uint);
-    function buy(uint _amount, string _sku, address _addr_appc, address _dev, address _appstore, address _oem) public view returns (bool);
+    function buy(uint _amount, string _sku, address _addr_appc, address _dev, address _appstore, address _oem, string packageName, bytes2 countryCode) public view returns (bool);
```
Comment (tcrosado) [inline contracts/AppCoinsIAB.sol]: It's a minor thing but, wouldn't it be better to have package name and _sku as the first arguments of the function and the event? Or at least packageName near sku, as they are related.
Another minor detail,  packageName and countryCode should both start with underscore, for consistency.

Comment (GustavoRSSilva) [reply contracts/AppCoinsIAB.sol]: good point.

**Review by tcrosado — APPROVED** [review]

**Review by DioPires — APPROVED** [review]

**Pull request merged**